nodestream_akamai/gtm.yaml

```diff
@@ -37,4 +37,11 @@
         do_remove_trailing_dots: true
       node_key:
         target: !jmespath 'trafficTargets[*]'
-      find_many: true
\ No newline at end of file
+      find_many: true
+     - type: relationship
+      node_type: Endpoint
+      node_key: 
+        fqdn: !jmespath 'fqdn'
+      find_many: true
+      relationship_type: CONNECTS_TO
```

Comment:
@zprobst do you have an opinion on the RELATIONSHIP name here?

Reply:
I'd like to understand the domain better. So this relationship is `(:Endpoint)-[:CONNECTS_TO]->(:AkamaiGtmDomain)`. Does the proxy to it - in the sense that the endpoint traffic from this endpoint "ends up" at GTM? In which case I'd recommend `PROXIES_TO`.

Reply:
I will let @stuartio answer, but these Endpoints are actually hosted from Akamai DNS and they are typically A records.

Here is an example:
AkamaiGtmDomain.name = gbx.example.com.akadns.net
AkamaiGtmProperty.name = frontdoor.example.ca
AkamaiGtmProperty.fqdn = frontdoor.example.ca.gbx.example.com.akadns.net
Endpoint.fqdn = frontdoor.example.ca.gbx.example.com.akadns.net

```
; <<>> DiG 9.10.6 <<>> frontdoor.example.ca.gbd.example.com.akadns.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 62330
;; flags: qr rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;frontdoor.example.ca.gbd.example.com.akadns.net. IN A

;; ANSWER SECTION:
frontdoor.example.ca.gbx.example.com.akadns.net. 30 IN CNAME ww2.frontdoor.example.ca.gbx.example.com.akadns.net.
ww2.frontdoor.example.ca.gbx.example.com.akadns.net. 30 IN A xx.xx.xx.193
ww2.frontdoor.example.ca.gbx.example.com.akadns.net. 30 IN A xx.xx.xx.170
ww2.frontdoor.example.ca.gbx.example.com.akadns.net. 30 IN A xx.xx.xx.290

;; Query time: 195 msec
;; SERVER: 172.18.52.141#53(172.18.52.141)
;; WHEN: Wed Feb 07 14:44:21 PST 2024
;; MSG SIZE  rcvd: 139

```

Reply:
Seems like this aligns with a domain and a record, how about: `(:Endpoint)-[:RECORD_OF]->(:AkamaiGtmDomain)`

Reply:
Seems like this aligns with a domain and a record how about: `(:Endpoint)-[:RECORD_OF]->(:AkamaiGtmDomain)` or `(:Endpoint)<-[:HAS_RECORD]-(:AkamaiGtmDomain)`

Reply:
Yeah GTM and DNS are pretty much equivalent, at least structurally. So RECORD_OF is fine if you would prefer

Reply:
We agreed to move forward with RECORD_OF.  Thanks @stuartio